src/fides/api/models/privacy_notice.py

```diff
@@ -324,6 +335,28 @@ def systems_applicable(self) -> bool:
                 return True
         return False
 
+    @property
+    def configured_regions(self) -> List[PrivacyNoticeRegion]:
+        """Convenience property to look up which regions are using these Notices."""
+        from fides.api.models.privacy_experience import (
+            ExperienceNotices,
+            PrivacyExperience,
+        )
+
+        db = Session.object_session(self)
+        configured_regions = (
+            db.query(PrivacyExperience.region)
+            .join(
+                ExperienceNotices,
+                PrivacyExperience.experience_config_id
+                == ExperienceNotices.experience_config_id,
+            )
+            .filter(ExperienceNotices.notice_id == self.id)
+            .group_by(PrivacyExperience.region)
+            .order_by(PrivacyExperience.region.asc())
+        )
+        return [region[0] for region in configured_regions]
+
```

Comment:
This is the property that is used to build the "configured" attribute in the Privacy Notice List view, useful for displaying the number of locations using it.  Previously we could just read the regions straight off the Notice, but it is no longer stored there.

Reply:
I've tried to mitigate by only exposing this property on the Privacy Notice List and Detail endpoints used in the Admin UI, not for the end user, and also further pulled a lot of the details previously exposed in Privacy Notice API endpoint to help that load.

I agree there's always tradeoffs here.  Model level properties that you then just expose in the API are convenient and readable, but yes defined like this, certainly less performant than it could be.  I'll make a note to revisit this with the locations PR.